### PR TITLE
form with  mixin

### PR DESCRIPTION
**Pull request by sansadev**

### CODE REVIEW

**Comment by cherimarie** [comment]
@sansadev when I run this code locally, there is no animation or interactivity on the page. Please review your code and let me know when it's ready for grading.

**Comment by sansadev** [comment]
hi cheri,
i noticed its all gone from my folder.
but its der in github
can u check this link?
https://github.com/sansadev/html300-v2/pull/4/commits/16fc7412ed4986a2b4c63c65d7d927f2adb22efd
i am trying to copy from there.

Thanks
swati

**Comment by cherimarie** [comment]
@sansadev Tyler let me know there is an animation at smaller screen sizes. I'm able to see this when running your code locally. In future pull requests, please make clear in the pull request description what is changing, and what your instructor should be looking for to grade (or your coworker should be looking at to review).

File: lesson04/assignment/css/_nav.scss [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+     z-index:2;
+   }
+   //use mask
+   .nav-links {
+     position:fixed;
+     background: $nav;
+     height: 100vh;
+     width: 100%;
+     flex-direction: column;
+     clip-path:circle(100px at 80% -20%);
```
Comment (cherimarie) [inline lesson04/assignment/css/_nav.scss:63]: Nice!

File: lesson04/assignment/css/_nav.scss [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+     transition: all 1s ease-out;
+
+   }
+   //creating a class called .open to use in jabvascript to animate
```
Comment (cherimarie) [inline lesson04/assignment/css/_nav.scss:67]: Typo: "jabvascript" -> "javascript"

File: lesson04/assignment/css/style.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+* {
+  box-sizing: border-box;
+  margin:0;
+  padding:0;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+
+}
```
Comment (cherimarie) [inline lesson04/assignment/css/style.scss:16]: Don't forget to import _variables.scss, or just delete that file if it's not needed.

File: lesson04/assignment/index.html [hunk 4]
```diff
@@ -5,11 +5,36 @@
...
+</div>
+
+
+
+
+
+
+
+  <script src="app.js"></script>
 </body>
```
Comment (cherimarie) [inline lesson04/assignment/index.html:39]: I'm not seeing any of the jpg or gif images used in this file. They shouldn't be included in the repo if they're not used. We only want necessary files in our repo.

**Pull request merged**